htdocs/contrat/list.php

```diff
@@ -201,7 +201,7 @@
 $contracttmp = new Contrat($db);
 
 $sql = 'SELECT';
-$sql .= " c.rowid, c.ref, c.datec as date_creation, c.tms as date_update, c.date_contrat, c.statut, c.ref_customer, c.ref_supplier, c.note_private, c.note_public,";
+$sql .= " c.rowid, c.ref, c.datec as date_creation, c.tms as date_update, c.date_contrat, c.statut, c.ref_customer, c.ref_supplier, c.note_private, c.note_public, c.entity";
```

Comment:
If you add a new field into the select, you must also add it into the group by. Can you do this ?

Reply:
done in fbeac7c3742c2028c8d7217326600d6e7bf34a3b